department-of-veterans-affairs/caseflow
# Disabled button after user clicks

Connects #6534 

### Description
This PR disables the 'continue' button on the 'Upload Schedule Period' page after the user clicks.

### Testing Plan
1. Go to 'hearings/schedule/build/upload'
1. Click continue, confirm errors show up as expected and button is enabled after the errors show up
1. Select either Judge or RO, click continue, confirm the same as above
1. Select an invalid spreadsheet, click continue, confirm the button is disabled as the request to the backend is processed and is reenabled once we get a response
1. Select a valid spreadsheet, click continue, confirm the button is disabled as the request is processed, when redirected, click 'Go back', and confirm the button is reenabled
